app/src/main/java/technopark/andruxa/myapplication/network/album/retrofit2/AlbumRetrofit.kt

```diff
@@ -0,0 +1,52 @@
+package technopark.andruxa.myapplication.network.album.retrofit2
+
+import com.tickaroo.tikxml.TikXml
+import com.tickaroo.tikxml.retrofit.TikXmlConverterFactory
+import okhttp3.HttpUrl
+import okhttp3.OkHttpClient
+import retrofit2.Call
+import retrofit2.Retrofit
+import retrofit2.http.GET
+import retrofit2.http.Query
+import technopark.andruxa.myapplication.models.album.AlbumSearchResponse
+import technopark.andruxa.myapplication.models.album.XML.AlbumSearchResponseXML
+import technopark.andruxa.myapplication.network.album.AlbumApi
+
+class AlbumRetrofit: AlbumApi {
```

Comment:
`RetrofitAlbum` and `AlbumRetrofit` are very similar names and looks quite confusing
Actually I think you don't need additional wrapper around Retrofit interface